acrobat/scripts/scripts.js

```diff
@@ -200,9 +200,26 @@ const CONFIG = {
   if (STYLES) { paths.push(STYLES); }
   loadStyles(paths);
 
+  // Labeling the placeholders before they get processed by Milo
+  const placeholderLabel = (document) => {
+    const ccregex = /{{(.*?)}}/g;
+    const matches = [...document.innerHTML.matchAll(ccregex)];
+    if (matches.length > 0) {
+      const pTags = document.querySelectorAll('p');
+      pTags.forEach((pTag) => {
+        if (pTag.innerHTML.match(ccregex)) {
+          const translationLabel = pTag.innerHTML;
+          const label = translationLabel.slice(2, -2);
+          pTag.setAttribute('data-local', label);
+        }
+      });
+    }
+  };
+  placeholderLabel(document.documentElement);
```

Comment:
might consider doing this the other way around to see if faster, but again I really like this approach

Reply:
Not sure if it matters, but does this need to happen before loadArea?